docs/Dual_Carriage.md

```diff
@@ -0,0 +1,98 @@
+This document provides **general** information on configuring and
+calibrating a standard IDEX printer. Klipper has support for very
+complex IDEX configurations, with multiple hotends and extruders on a
+single carriage, independent or grouped cooling fans, and much more,
+which are not covered in this document.
+
+# General configuration
+Start with studying, and possibly using parts of the [example config](../config/sample-idex.cfg)
+for IDEX printers. It has some distinct items that you may or may not
+wish to use.
+* Definition for `dual_carriage` on the X-axis, which you can change to
+be on the Y axis if that is how your printer is configured.
+* Sample macros for parking and switching toolheads. You should check
+these to make sure they make sense for your setup and update the
+position values with those that match your printer
+* Configuration for multiple part cooling fans. The `[fan]` module in
+Klipper only supports a single controllable fan. If you would like to
+have independently controlled part cooling fans, do not configure a
+`[fan]` section and use the template provided in the file.
+* A macro for calibrating carriage separation described later in this
+document.
+
```

Comment:
can you tell me more? I was actually wondering earlier if senseless homing would be possible. For example you home the left extruder and then you home the right extruder into the left extruder. This would solve the endstop_phase issue I'm investigating

Reply:
I tried sensorless homing on my custom printer, and I just couldn't get any consistency better than ~0.2mm. which is extremely noticeable on the alignment prints. I'd get it lined up, then on the next home it would not line up anymore.

In this nice doc: https://duet3d.dozuki.com/Wiki/Stall_detection_and_sensorless_homing#Section_Limitations_of_stall_detection
Note the +-2 full step accuracy. Multiply this variance by 2 (one for each side) and you see.

Part of the inaccuracy (as I understand it) is that the EM feedback changes depending on motor coil heat. So cold motors, once heated up, seems to home less sensitively.

I suppose some motors/setup might achieve acceptable accuracy, but my repurposed A8 motors have a very high inductance, and is more prone to variations. (as I understand it)

Reply:
Oh have them all home against each other? Hmm, could work...
But I have started crimping wires for microswitch endstops, so will probably use those with phase assistance.